index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Video Page for Microvers Warm Up Project</title>
+  <link href="styles.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+
+  <header>
+  <nav>
+<ul>
+  <li>
+<a href="#">Home</a>
+  </li>
+ <li>
+<a href="#">About</a>
+  </li>
+ <li>
+<a href="#">Contact</a>
+  </li>
+ <li>
+<a href="#">Terms of Use</a>
+  </li>
+ <li>
+<a href="#">Join Us</a>
+  </li>
+</ul>
+</nav>
+</header>
+
+ <h2 class="center"  class="h2">Welcome to Dario's Video Page for Microverse Warm Up Project</h2> 
+  
+  <p class="center">Here you can see beautiful videos of one of Venezuela's greatest landmarks</p>
+  
+<iframe class="videoborder" max-width="350" height="auto"  src="https://www.youtube-nocookie.com/embed/DIz_i9WNgr8" title="YouTube video player" frameborder="10" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+ <!-- Sidebar -->
+<div class="border" class="posi">
+  <h5 class="divside">Roraima's Playlist</h5>
+<iframe class="divside" width ="90" height="56" src="https://www.youtube.com/embed/videoseries?list=PLK8RSidnF7Lfdk8pLwUZ7ExI4SSt1jWdH&rel=0&loop=1" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-inpicture" allowfullscreen></iframe>
+</div>
```

Comment:
Kindly use tags such as main, article, or section tags to make your code more SEO friendly